.github/workflows/wait-for-akd-publish.sh

```diff
@@ -0,0 +1,46 @@
+# Finds expected version of a crate in another crate's Cargo.toml file
+get_crate_expected_version_number()
+{
+    local crate_name=$1
+    local crate_to_look_for_version=$2
+
+    local cargo_toml_file="$crate_name/Cargo.toml"
+    local expected_version=$(grep "$crate_to_look_for_version = " $cargo_toml_file | tr -d '{^}' | awk -F '[,:=]' '{print $5}')
+    echo $expected_version
+}
+
+# Get published versions of a crate from https://github.com/rust-lang/crates.io-index/
+get_crate_published_versions()
+{
+    local crate_index_url=$1
+
+    local published_versions=$(
+        curl -sS "$crate_index_url" | jq .vers
+    )
+    echo "$published_versions"
+}
+
+# Wait for a specific akd version to be published to crates.io.
+# See https://github.com/novifinancial/akd/issues/116.
+# Must be run in the project root folder.
+akd_version_expected=$(get_crate_expected_version_number "akd_mysql" "akd")
+echo "AKD expected version in AKD_MYSQL:" $akd_version_expected
+wait_time=1
+while sleep $wait_time;
+do
+    akd_versions_published=$(
+        get_crate_published_versions "https://raw.githubusercontent.com/rust-lang/crates.io-index/master/3/a/akd"
+    )
+    echo "AKD published versions:" $akd_versions_published
+    # Check whether published versions contain the expected version
+    if [[ $akd_versions_published == *"$akd_version_expected"* ]]; then
+        echo "Expected version has been published."
+        break
+    fi
+    echo "Expected version has not been published. Retrying after a wait."
+    wait_time=$((wait_time+1))
+    if [[ $wait_time == 42 ]]; then
```

Comment:
Uhm... why 42? lol

Reply:
30 seconds gives ~8 minutes for AKD to be published before giving up. Publishing AKD took ~6 minutes (to my surprise) in one instance.

For 42, ~15 minutes should be enough to make sure AKD is published (I think... but I'm not sure of the variation here). Why exactly 42? Somewhat arbitrary and [possibly](https://www.scientificamerican.com/article/for-math-fans-a-hitchhikers-guide-to-the-number-42/) not...